devel/docker/settings/policy.yaml

```diff
@@ -0,0 +1,147 @@
+  - !PassingTestCaseRule {test_case_name: compose.install_scsi_updates_img, scenario: "fedora.universal.x86_64.64bit"}
+  - !PassingTestCaseRule {test_case_name: compose.realmd_join_sssd, scenario: "fedora.Server-dvd-iso.x86_64.64bit"}
+  - !PassingTestCaseRule {test_case_name: compose.server_cockpit_default, scenario: "fedora.Server-dvd-iso.x86_64.64bit"}
+  - !PassingTestCaseRule {test_case_name: compose.server_database_client, scenario: "fedora.Server-dvd-iso.x86_64.64bit"}
+  - !PassingTestCaseRule {test_case_name: compose.server_firewall_default, scenario: "fedora.Server-dvd-iso.x86_64.64bit"}
+  - !PassingTestCaseRule {test_case_name: compose.server_realmd_join_kickstart, scenario: "fedora.Server-dvd-iso.x86_64.64bit"}
+  - !PassingTestCaseRule {test_case_name: compose.server_role_deploy_database_server, scenario: "fedora.Server-dvd-iso.x86_64.64bit"}
+  - !PassingTestCaseRule {test_case_name: compose.server_role_deploy_domain_controller, scenario: "fedora.Server-dvd-iso.x86_64.64bit"}
+
+--- !Policy
+id: "taskotron_release_critical_tasks_for_testing"
+product_versions:
+  - fedora-31
+  - fedora-30
+  - fedora-29
+  - fedora-28
+decision_context: bodhi_update_push_testing
+blacklist: []
+subject_type: koji_build
+rules:
+  - !RemoteRule {}
+
+--- !Policy
+id: "taskotron_release_critical_tasks_for_stable"
+product_versions:
+  - fedora-30
+  - fedora-29
+  - fedora-28
+  - fedora-27
+  - fedora-26
+decision_context: bodhi_update_push_stable
+blacklist: []
+subject_type: koji_build
+rules:
+  - !RemoteRule {}
+
+--- !Policy
+id: "no_requirements_testing"
+product_versions:
+  - fedora-30-modular
+  - fedora-30-containers
+  - fedora-30-flatpaks
+  - fedora-29-modular
+  - fedora-29-containers
+  - fedora-29-flatpaks
+  - fedora-28-modular
+  - fedora-28-containers
+  - fedora-epel-7
+  - fedora-epel-6
+decision_context: bodhi_update_push_testing
+blacklist: []
+subject_type: koji_build
+rules: []
+
+--- !Policy
+id: "no_requirements_for_stable"
+product_versions:
+  - fedora-30-modular
+  - fedora-30-containers
+  - fedora-30-flatpaks
+  - fedora-29-modular
+  - fedora-29-containers
+  - fedora-29-flatpaks
+  - fedora-28-modular
+  - fedora-28-containers
+  - fedora-epel-7
+  - fedora-epel-6
+decision_context: bodhi_update_push_stable
+blacklist: []
+subject_type: koji_build
+rules: []
+
+--- !Policy
+# openQA policies
+id: "openqa_release_critical_tasks_for_testing"
+product_versions:
+  - fedora-30
+  - fedora-29
+  - fedora-28
+  - fedora-27
+  - fedora-26
+decision_context: bodhi_update_push_testing
+blacklist: []
+subject_type: bodhi_update
+rules:
+  - !RemoteRule {}
+
+--- !Policy
+id: "openqa_release_critical_tasks_for_stable"
+product_versions:
+  - fedora-30
+  - fedora-29
+  - fedora-28
+  - fedora-27
+  - fedora-26
+decision_context: bodhi_update_push_stable
+blacklist: []
+subject_type: bodhi_update
+rules:
+  - !RemoteRule {}
```

Comment:
I bet we could also download this file from Fedora infra upon vagrant up, so that we don't have to maintain our own copy. In fact, we also have a copy of this for our CI tests - it would probably be best if that file were also downloaded at test run time.

Reply:
I noticed that this file is a bit outdated in CI so decided to create new one. But idea with auto-download for docker and CI is a good idea.

Reply:
Looks to me like this is resolved.